### PR TITLE
Adding Micah Gale

### PR DESCRIPTION
**Pull request by MicahGale**
Did a quick add of myself.

This will need clarification once I actually have a more concrete research topic.

### CODE REVIEW

File: community/people/mgale/DSC_4522.jpg
Comment (gonuke) [inline community/people/mgale/DSC_4522.jpg:1]: The template forces this to a square aspect ratio, so best to crop it to something nearly square.

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for pulling this together @MicahGale - just one quick comment on the photo size.

**Review by lewisgross1296 — COMMENTED** [review]
This LGTM. One thing you may want to do to clean up the website commit history is an interactive rebase. If you've never done this before, essentially you squash all your changes into one commit. Locally, you'd
```
git rebase -i HEAD~N
```
where N is the number of commits that you'll squash together. It seems like here you'd do N=4. Once that is done, it will allow you to decide which commits to pick or squash and then you can add a description for the commit that will be in the history. 

Since you rebase locally and the commit history on the PR doesn't match your local commit history, you'll need to force push after (git push -f BRANCH). Feel free to slack me if this seems unclear.

File: community/people/mgale/DSC_4522.jpg
Comment (MicahGale) [reply community/people/mgale/DSC_4522.jpg:1]: Switched to a square crop with ff6bfeb3282.

**Comment by MicahGale** [comment]
> 

Generally I am not a fan of rewriting history in git, and force pushes are generally not the solution.

However, given that I committed a jpg twice, and we aren't using LFS I'll squash to be nice to everyone else.

**Comment by MicahGale** [comment]
I felt like being on the bleeding edge: so now this depends on #209, and should wait until that is merged first.

**Comment by gonuke** [comment]
#209 should be ready for review/merge

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
Does this need a rebase after merging #209?

**Comment by MicahGale** [comment]
It seems github is ok with a simple merge. I'll do a quick merge locally just to be sure.

**Comment by gonuke** [comment]
This looks great now.

**Pull request merged**
